app/views/provider_interface/decisions/new_defer_offer.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @defer_offer, url: provider_interface_application_choice_defer_offer_path(@application_choice) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <span class="govuk-caption-xl">
+        <%= @application_choice.application_form.full_name %>
+      </span>
+
+      <h1 class="govuk-heading-xl">
+        Check and confirm offer deferral
+      </h1>
+
+      <p class="govuk-body">This application will be marked as deferred.</p>
+      <p class="govuk-body">You must review and confirm your offer again at the start of the next cycle (<%= RecruitmentCycle.years_in_next_cycle.join(' to ') %>).</p>
```

Comment:
I'd almost rather have `RecruitmentCycle#next_cycle_name` (returning `2021 to 2022`) than `RecruitmentCycle#years_in_next_cycle.join(' to ')` but I'm not sure which is simpler. We could make the years dynamic in any case using `RecruitmentCycle.next_year`?